Reference/Routing/custom-controllers.md

```diff
@@ -206,3 +287,4 @@ public class MyRenderMvcController : IRenderMvcController
     }
 }
 ```
+or inherit from the default RenderMVCController as in the route hijacking examples.
```

Comment:
```suggestion
It can also be achieved by inheriting from the default RenderMVCController as in the route hijacking examples.
```

Reply:
This can also be achieved.. 
rather than It ?

Reply:
Yes,I like "This can also be achieved..." better!

Reply:
Have simplified this further so the text is split across the code example.